driver-async/src/test/functional/com/mongodb/async/client/TransactionsTest.java

```diff
@@ -162,7 +160,9 @@ public void apply(final SocketSettings.Builder builder) {
                 .writeConcern(getWriteConcern(clientOptions))
                 .readConcern(getReadConcern(clientOptions))
                 .readPreference(getReadPreference(clientOptions))
-                .build());
+                .retryWrites(clientOptions.getBoolean("retryWrites", BsonBoolean.FALSE).getValue());
```

Comment:
Should the spec tests be updated to explictly set this to false?

Reply:
I think not: See the discussion on https://github.com/mongodb/specifications/pull/522.  Java driver is actually out of spec in its use of exact matching of command events.

Reply:
ACK